spec/features/service_providers_spec.rb

```diff
@@ -197,6 +197,18 @@
       click_on t('forms.buttons.trigger_idp_refresh')
       expect(page).to have_content(I18n.t('notices.service_providers_refreshed'))
     end
+
+    scenario 'can enable prompt=login for a service provider' do
+      admin = create(:admin)
+      sp = create(:service_provider, :with_team)
+      login_as(admin)
+
+      visit edit_service_provider_path(sp)
+      check 'Enabled'
```

Comment:
Suggestion...
Should we address potential conflicts with other checkbox's added to this page?

Should we try 
```
check 'allow_prompt_login'
```
And reference the ID of the field? If `Enabled` is the field ID, then maybe we should change it to avoid elements with the same id.

Reply:
Yup makes sense, that was just the label so I updated the test.